src/Types/TypeRegistry.php

```diff
@@ -65,6 +71,8 @@ public function has(string $name): bool
     /**
      * Registers a custom type to the type map.
      *
+     * @psalm-param non-empty-string $name
```

Comment:
Something's not sound here. If passing an empty string is invalid, shouldn't the method throw in that case? Theoretically, what would happen if I registered a type under the name `''`? Would that work?

Reply:
Yes @derrabus. It is currently working without exceptions. IE:
```patch
--- a/tests/Types/TypeRegistryTest.php
+++ b/tests/Types/TypeRegistryTest.php
@@ -81,6 +81,16 @@ class TypeRegistryTest extends TestCase
         self::assertSame($newType, $this->registry->get('some'));
     }
 
+    public function testRegisterEmptyName(): void
+    {
+        $newType = new TextType();
+
+        $this->registry->register('', $newType);
+
+        self::assertTrue($this->registry->has(''));
+        self::assertSame($newType, $this->registry->get(''));
+    }
+
     public function testRegisterWithAlreadyRegisteredName(): void
     {
         $this->registry->register('some', new TextType());
```